edgelet/edgelet-docker/src/lib.rs

```diff
@@ -13,9 +13,29 @@
 
 // mod client;
 mod error;
+mod image_prune_data;
```

Comment:
Can the contents of image_prune_data.rs be moved to a new image-gc crate along with the contents of aziot-edged/image_gc.rs (see my earlier comment in image_gc.rs)?